<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-8UKHKL7O" failures="1" tests="4" name="BankingApp Test" time="12.362" errors="0" timestamp="2024-10-03T15:30:33 IST">
  <testcase classname="com.bank.tests.LoginTest" name="loginWithValidCredentials" time="2.631"/>
  <testcase classname="com.bank.tests.LoginTest" name="loginWithInvalidCredentials" time="1.816"/>
  <testcase classname="com.bank.tests.LoginTest" name="failure" time="0.004">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.bank.tests.LoginTest.failure(LoginTest.java:50)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- failure -->
  <testcase classname="com.bank.tests.LoginTest" name="skipped" time="0.0">
    <skipped/>
  </testcase> <!-- skipped -->
</testsuite> <!-- BankingApp Test -->
